### ISSUE TITLE
Relative imports

### ISSUE THREAD

**Issue by adamerose** [satwikkansal/wtfpython#240]
I definitely think this repo should include an example about how messed up relative imports are in Python, just read this thread https://stackoverflow.com/questions/16981921/relative-imports-in-python-3

**Comment by satwikkansal** [comment]
Yes, imports are definitely a missing part of this collection. Will add this in the next iteration for sure.  

Also related: https://github.com/satwikkansal/wtfpython/issues/133
